other-requirements.txt

```diff
@@ -1,17 +0,0 @@
-# runtime dependencies
-python-gobject
-python-gi-cairo
-gir1.2-gtk-3.0
-
-# build dependencies
-python-docutils  # for rst2man
-libglib2.0-bin   # for glib-compile-schemas
-gettext          # for msgfmt
-
-# test dependencies (in a clean VM, so you have GTK+ themes etc.)
-dbus-x11
-gnome-themes-standard
-gnome-icon-theme-full
-gnome-icon-theme-symbolic
-libcanberra-gtk3-module
-gedit
```

Comment:
I'm not sure how I feel about removing this file.  It's an input file for [bindep](https://pypi.org/project/bindep/), which can check if your system has these packages installed.

I was very enthusiastic about bindep when it first appeared on the scene, but I haven't actually gotten any use out of it.  (I didn't even notice when it changed the default input filename from other-requirements.txt to bindep.txt.)  Maybe it's not actually useful.

Reply:
The other day I was testing in a minimal Ubuntu 16.04 container and I found other-requirements.txt very helpful.

(I did not install everything from it, but I did find the package names of the gir1.2-* things useful.)